src/controllers/update/confirm.overseas.entity.details.controller.ts

```diff
@@ -15,12 +16,21 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
     const appData: ApplicationData = getApplicationData(req.session as Session);
     const update = appData.update as Update;
 
+    let journeyQuery = "";
```

Comment:
Please follow the pattern used on other screens - so if a remove journey, have a new `res.render` command in the conditional block and leave the existing one unchanged.

Reply:
Updated this.

Reply:
Thanks.